src/components/careers/Positions/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { Grid } from '@material-ui/core'
+import styled from 'styled-components'
+
+import { Card } from './Card'
+import ContentWrapper from '../../Layout/ContentWrapper'
+
+const JOBS_API_URL = 'https://boards-api.greenhouse.io/v1/boards/gnosis/jobs'
+
+const SectionTitle = styled.h2`
+  font-size: 45px;
+  font-weight: bolder;
+  text-align: center;
+  margin-bottom: 56px;
+`
+
+type JobMeta = {
+  id: number
+  name: string
+  value: unknown
+  value_type: string
+}
+
+type Job = {
+  absolute_url: string
+  data_compliance: unknown[]
+  id: number
+  internal_job_id: number
+  location: {
+    name: string
+  }
+  metadata: JobMeta[]
+  requisition_id: unknown
+  title: string
+  updated_at: string
+}
+
+type Jobs = {
+  jobs: Job[]
+  meta: {
+    total: number
+  }
+}
+
+export const Positions = () => {
+  const [positions, setPositions] = useState<Jobs>()
+
+  useEffect(() => {
+    let isCurrent = true
+
+    const getOpenPositions = async () => {
+      try {
+        const data = await fetch(JOBS_API_URL)
+        const jobs: Jobs = await data.json()
+
+        if (isCurrent) {
+          setPositions(jobs)
+        }
+      } catch (e) {
+        console.error(e)
+      }
+    }
+
+    getOpenPositions()
+
+    return () => {
+      isCurrent = false
+    }
+  }, [])
+
+  return (
+    <ContentWrapper>
+      <SectionTitle>Open positions</SectionTitle>
+      <Grid container spacing={3}>
+        {positions &&
```

Comment:
What if there are no open positions?

Reply:
We could either display a message or hide the section altogether.

Reply:
Let's hide it, for simplicity's sake.

Reply:
WDYT about adding a message like "Currently we don't have any open positions, make sure to come back later". Also, sometimes companies have an "Open application" position in case there's no suitable position for someone, but they're really keen on working at the company, perhaps we also want to add this

Reply:
I like the idea of displaying a message if there are no open positions, gives the user more information and could avoid confusion.